filesystem/android_super.ksy

```diff
@@ -0,0 +1,207 @@
+meta:
+  id: android_super
+  title: Android Dynamic Partitions metadata
+  application: Android
+  file-extension: img
+  license: CC0-1.0
+  ks-version: 0.9
+  bit-endian: le
+  endian: le
+
+doc: |
+  The metadata stored by Android at the beginning of a "super" partition, which
+  is what it calls a disk partition that holds one or more Dynamic Partitions.
+  Dynamic Partitions do more or less the same thing that LVM does on Linux,
+  allowing Android to map ranges of non-contiguous extents to a single logical
+  device. This metadata holds that mapping.
+
+doc-ref:
+  - https://source.android.com/devices/tech/ota/dynamic_partitions
+  - https://android.googlesource.com/platform/system/core/+/master/fs_mgr/liblp/include/liblp/metadata_format.h
```

Comment:
It's generally not a good idea to link to the `master` branch (or any branch that _changes_ over time) when you refer to any source code, because, you know: software evolves and code gets refactored and regrouped, old code is deleted etc.

Please try to link a specific tag or a commit hash.

Reply:
Done